### DISCUSSION TITLE
Signal

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#135] [Ideas]
Hi again ! Would you be able to add a '' send signal '' option where you can type anything you want in it ? :)

**Comment by winstxnhdw** [comment]
Do you mean the Terminal signal?

**Reply by Gloktober** [reply]
yes exactly !

**Reply by winstxnhdw** [reply]
Sure, will get it done in a minute.

**Reply by Gloktober** [reply]
crazy how fast you are lol . Thank you

**Reply by winstxnhdw** [reply]
Done.

**Reply by winstxnhdw** [reply]
It's `/signal <message>`

**Reply by winstxnhdw** [reply]
There seems to be a hard check for messages greater than 12 characters long :/

**Reply by Gloktober** [reply]
it's okay ! though , the command does not work for me

**Reply by winstxnhdw** [reply]
I think you must have a `Signal Translator` first.

**Reply by winstxnhdw** [reply]
Okay, I made sure the command now spawns a Signal Transmitter.

**Reply by Gloktober** [reply]
correct ! thank you.
